ppx/ppx_tyxml.ml

```diff
@@ -192,30 +266,8 @@ let ast_to_stream expr =
       (Antiquot.create expr, expr.pexp_loc.loc_start)
   in
 
-  let items = ref strings in
-  let offset = ref 0 in
-  let consumed = ref 0 in
-
-  let rec next () = match !items with
-    | [] -> None
-    | (s, loc)::rest ->
-      if !offset = 0 then begin
-        current_adjust_location := Loc.adjust loc !consumed;
-        consumed := !consumed + String.length s
-      end;
-
-      if !offset < String.length s then begin
-        offset := !offset + 1;
-        Some (s.[!offset - 1])
-      end
-      else begin
-        offset := 0;
-        items := rest;
-        next ()
-      end
-  in
-
-  Markup.fn next, (fun x -> !current_adjust_location x)
+  Markup.string (String.concat "" (List.map fst strings)),
```

Comment:
> except in case someone pastes several megabytes of markup into a PPX quotation

Pretty sure that's going to happen at some point when I add the converter `.html/.mustache -> ocaml` ....